packages/mongodb/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@1.9
```

Is there any reason to don't use 1.10?
```suggestion
    reference: git@1.10
```

---

No, we can go with v1.10. Fixed.